.github/workflows/e2e.yml

```diff
@@ -89,11 +89,15 @@ jobs:
         if: ${{ matrix.dataset }}
         run: python -c "${{ matrix.dataset }}"
       - name: Run edge client test
-        run: ./../test.sh "${{ matrix.directory }}"
-      - name: Run virtual client test
+        run: ./../test_server.sh "${{ matrix.directory }}"
+      - name: Run virtual client test 
         run: python simulation.py
-      - name: Run driver test
-        run: ./../test_driver.sh
+      - name: Run driver test using gRPC bidi and in-memory state
+        run: ./../test_flower_server.sh bidi mem
+      - name: Run driver test using gRPC bidi and SQLite database
```

Comment:
@charlesbvll could you elaborate on why the `test_server.sh` needs the directory to be passed but the subsequent scripts don't? I think we should make this more uniform.

Reply:
I made the change to make it more uniform